Automattic/wp-calypso
# Happychat: Cleanup unnecessary transcript timeout

#### Changes proposed in this Pull Request

This PR is a follow-up cleanup to https://github.com/Automattic/wp-calypso/pull/59958#discussion_r783847094 where @jsnajdr discovered that the happychat transcript timeout action does nothing and can be cleaned up. This PR takes the cleanup a step further and deletes the unnecessary action creator and timeout functionality from the happychat client.

#### Testing instructions

* Enable yourself as an agent with available chat slots in the Happychat staging HUD.
* Open the chat in Calypso.
* Test sending and receiving messages.
* Reload and verify the chat session persists.
* Smoke test happychat and verify everything still works well.
* Keep an eye on the console for errors
* Verify tests still pass.
